wire-ios-request-strategy/Sources/Request Strategies/Connection/ConnectionRequestStrategy.swift

```diff
@@ -203,6 +209,20 @@ extension ConnectionRequestStrategy: ZMEventConsumer {
                         conversationEvent,
                         in: managedObjectContext
                     )
+
+                    if conversationEvent.connection.status == .accepted, let conversationID = conversationEvent.connection.qualifiedTo {
+
+                        WaitingGroupTask(context: managedObjectContext) { [self] in
+                            do {
```

Comment:
suggestion: add a comment explaining why we add a delay.

Reply:
Suggestion: The client who accepts the connection resolves the conversation immediately. Other clients (from self and other user) resolve after a delay to avoid a race condition, but also to re-attempt resolution in case of failure.

Reply:
Done!